src/main/java/com/google/sps/server/AddSchedule.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2010 Google Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.sps.server;
+
+import com.google.api.services.calendar.model.CalendarList;
+import com.google.api.services.calendar.model.CalendarListEntry;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.Events;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.api.client.util.DateTime;
+
+import java.util.Date;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collection;
+import java.util.Iterator;
+import java.io.IOException;
+
+import com.google.sps.data.*;
+
+class AddSchedule {
+   /**
+   * Puts schedule onto secondary calendar
+   * 
+   * @param client the user's Google calendar 
+   * @param calendarId the id of the secondary calendar to put the schedule on
+   */
+  public static void addSchedule(Schedule schedule, com.google.api.services.calendar.Calendar client, String calendarId) throws IOException {
+    try {
+      //get current date, so can add the courses to the appropriate days of the current month 
+      java.util.Calendar calendar = java.util.Calendar.getInstance();
+      int nextSun = calculateDateOfNextSunday(calendar);
```

Comment:
Small suggestion: can maybe rename `calculateDateOfNextSunday()` to `getNextSundayDate()` to be inline with the next 2 function names